src/cerberus.py

```diff
@@ -8,7 +8,6 @@
 import requests
 import _thread
 import logging
-from kubernetes.client.rest import ApiException
```

Comment:
ApiException definition is needed to catch the exception when the client is not able to get a response from the server.